### DISCUSSION TITLE
Not recognized liquid tag

### DISCUSSION THREAD

**Discussion by ypwang0371** [alshedivat/al-folio#388] [Q&A]
Hello, I found this error when I tried to use the template to build my personal website. Could anyone help me with this? Thank you so much!
![image](https://user-images.githubusercontent.com/89311305/132111129-ed4bf2b8-d5b4-4d88-885d-28d29b768d0b.png)

**Comment by JulianStreyczek** [comment]
Hi. In case you overlooked it, make sure to follow the [Getting Started](https://github.com/alshedivat/al-folio#getting-started) guide, especially the section "For personal and organization webpages". 

Also, some past issues (e.g. #315) might be related to your problem. Could you confirm if you manage to solve it?

**Reply by ypwang0371** [reply]
Hi Julian, thank you so much! I will explore it. I will let you know if I solve it.

**Reply by dxvidmr** [reply]
I got the same issue with the publications.md file and this solved it: go to your repo `Settings` and in `Pages` change the source to `gh-pages`. And save!
